### PR TITLE
Bumped dependencies

### PR DESCRIPTION
**Pull request by go-bears**

### CODE REVIEW

File: gui/package.json [hunk 1]
```diff
@@ -12,20 +12,25 @@
...
+    "babel": "^6.1.0",
+    "babel-core": "^6.1.0",
+    "babel-loader": "^6.1.0",
+    "babel-plugin-transform-decorators-legacy": "^1.3.4",
+    "babel-plugin-transform-object-rest-spread": "^6.1.0",
+    "babel-plugin-transform-runtime": "^6.1.0",
+    "babel-preset-es2015": "^6.1.0",
+    "babel-preset-react": "^6.1.0",
+    "babel-preset-stage-0": "^6.1.0",
+    "babel-register": "^6.1.0",
```
Comment (singingwolfboy) [inline gui/package.json:24]: Seems like you've added a few dependencies that weren't there before. Can you explain why?

Comment (go-bears) [reply gui/package.json:24]: I believe these were dependencies needed for setting up Jasmine testing for Travis CI. would that be the right summary @arbonap @minifast?

Comment (singingwolfboy) [reply gui/package.json:24]: I guess my question is, were these dependencies needed before, and weren't properly marked in the `package.json` file? Or did upgrading the existing dependencies require also pulling in these new dependencies?

Comment (go-bears) [reply gui/package.json:24]: @arbonap @minifast these dependencies weren't needed before Jasmine unit testing was introduced, yes?  To the best of my knowledge, these dependencies were added to "appease Travis" and to get Jasmine working properly.

**Comment by singingwolfboy** [comment]
I'm going to say that this pull request is ready to be merged. Thanks for doing this!

**Pull request merged**